scrum/tools/sprint_math.py

```diff
@@ -48,6 +48,46 @@ def request_call(url, have_headers):
     return response.json()
 
 
+def delete_card(id):
+    """Makes a DELETE request for the specified card using the requests library.
+
+    Args:
+        id: str, card id (not shortLink)
```

Comment:
Can you add an example? i.e `gGnafhow` vs `gGnafhow/2557-internal-work-removal-of-dead-link-in-specialty-admission` etc?

Reply:
updated